client/src/locales/fr-FR.json

```diff
@@ -433,10 +434,12 @@
         "documentation": "Documentation",
         "doc": "Doc.",
         "settings": "Paramètres",
+        "downloadParsec": "Télécharger Parsec pour ordinateur",
```

Comment:
Same here, `Télécharger l'application de bureau` is more explicit. It seems fitting to have the same text as the temporary suggestion.